docs/clients/voltage_superposition.rst

```diff
@@ -0,0 +1,142 @@
+    |variable_name| = iq.|module_name|(com, 0)
+    
+    zero_angle= |variable_name|.get("voltage_superposition", "zero_angle") 
+    print(f"Zero angle: {zero_angle}")
+
+Message Table
+~~~~~~~~~~~~~
+
+Type ID 74 | Voltage Superposition
+
++--------+-------------------------------+----------------+-----------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Sub ID | Short Name                    | Access         | Data Type | Unit          | Note                                                                                                                                                                                    |
++========+===============================+================+===========+===============+=========================================================================================================================================================================================+
+| 0      | zero_angle                    | get, set, save | float     | 1/s (rad)     | The angular position that the module considers as its 'zero' position to start pulsing.                                                                                                 |
++--------+-------------------------------+----------------+-----------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 1      | frequency                     | get, set, save | uint8     | Hz            | The number of pulses that happen per rotation.                                                                                                                                          |
++--------+-------------------------------+----------------+-----------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 2      | phase                         | get, set       | float     | 1/s (rad)     | This value is added to the zero angle to set where the pulse occurs.                                                                                                                    |
++--------+-------------------------------+----------------+-----------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 3      | amplitude                     | get, set       | float     | V             | This value represents the strength of the pulse. Setting the amplitude too high at lower speeds can cause the pulsing to overcome the inertia of the motor spinning, causing it to stop |
```

Comment:
I think we're replacing motor verbiage with module? @iq-fred?

Reply:
In the tutorials I stuck to module when describing the whole package. In this case I don't think its a big deal, it is the motor part of the module that is really spinning here. Not sure its worth changing up everything in this tables for it.